src/Middleware/Localization/src/RequestLocalizationOptions.cs

```diff
@@ -84,6 +84,12 @@ public RequestCulture DefaultRequestCulture
         /// </example>
         public bool FallBackToParentUICultures { get; set; } = true;
 
+        /// <summary>
+        /// Gets or sets the value of the reponse Content-Language header to a <see cref="CultureInfo.CurrentUICulture"/>.
+        /// Defaults to <c>true</c>
+        /// </summary>
+        public bool SetContentLanguageHeader { get; set; } = true;
```

Comment:
this would be a breaking change. I suggest to leave it as default as false. Also not really required for everyone.

Reply:
> this would be a breaking change.

No problem why it's a major release

> I suggest to leave it as default as false. Also not really required by everyone.

Agree

thanks a lot

Reply:
> No problem why it's a major release

FYI: A major release is **not** a free license to make breaking changes. We still work to evaluate breaking changes based on a cost/benefit analysis.

Reply:
You are right Andrew